.github/workflows/unittests.yml

```diff
@@ -0,0 +1,54 @@
+# For help debugging build failures open an issue on the RStudio community with the 'github-actions' tag.
+# https://community.rstudio.com/new-topic?category=Package%20development&tags=github-actions
+on: [push, pull_request]
+
+name: R-CMD-check
+
+jobs:
+  R-CMD-check:
+    runs-on: ${{ matrix.config.os }}
+
+    name: ${{ matrix.config.os }} (${{ matrix.config.r }})
+
+    strategy:
+      fail-fast: false
+      matrix:
+        config:
+          - {os: windows-latest, r: "3.6.1", jaspRequiredFilesBranch: "Windows"}
+          - {os: macOS-latest, r: "3.6.1",   jaspRequiredFilesBranch: "MacOS-Original"}
+          # - {os: ubuntu-20.04, r: "release", R_LIBS: "", rspm: "https://packagemanager.rstudio.com/cran/__linux__/focal/latest"}
+
+    env:
+      R_REMOTES_NO_ERRORS_FROM_WARNINGS: true
+      R_REMOTES_UPGRADE: never
+      VDIFFR_RUN_TESTS: true
+      REQUIRED_PKGS: ~/pkgs
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: r-lib/actions/setup-r@master
+        with:
+          r-version: ${{ matrix.config.r }}
+
+#      - uses: r-lib/actions/setup-pandoc@master
+
+      - name: Clone jasp-required-files
+        run: git clone --branch=${{ matrix.config.jaspRequiredFilesBranch }} https://github.com/jasp-stats/jasp-required-files.git ~/pkgs
+      
+      - name: Setup XQuarz
+        if : runner.os == 'macOS'
```

Comment:
I'd personally prefer `if: matrix.os == 'macOS-latest'` because then we're not using any variables that are not defined in this file